static/reference/next/pulsar-daemon/pulsar-daemon.md

```diff
@@ -7,6 +7,12 @@ pulsar-daemon has a similar interface to the pulsar command but adds start and s
 ### `start`
 Start a service in the background using nohup.
 
+:::tip
+
+You can use flags to perform more tasks. For example, `pulsar-daemon start standalone -nfw -nss` can start Pulsar faster in the background (without function workers and stream storage). `pulsar-daemon start standalone` flags are the same as `pulsar standalone` flags. For the complete flag list, see flags of the `pulsar standalone` command on the [Pulsar Reference Site](https://pulsar.apache.org/reference).
+
+:::
```

Comment:
Currently, this content can't be previewed through the normal preview steps (It might be caused by some technical issues. I'll record that in a GitHub issue). But it does not block merge this PR since the writing syntax is correct.